Python/auto_update_quarto.py

```diff
@@ -0,0 +1,35 @@
+"""
+This script will download and install the latest version of Quarto.
+Should work on Ubuntu >18.04.
+"""
+
+import os
+import time
+
+print("Upgrading Quarto")
+
+# Show the current quarto version
```

Comment:
Are there any constraints on how/where quarto was initially installed and should we document that?

Reply:
I'll add a note.